extras/jsonrpc/daemon.go

```diff
@@ -227,6 +227,7 @@ func (d *Client) ChannelList(account *string, page uint64, pageSize uint64) (*Ch
 		"page":             page,
 		"page_size":        pageSize,
 		"include_protobuf": true,
+		"wallet_id":        wid,
```

Comment:
this needs to be converted to a defined structure like in some other calls, otherwise I believe the SDK might crap out if you pass --wallet_id=null

Reply:
I covered it with a test and seems like the SDK handles it well